infra/aws-elb-503-subscription-endpoint/templates/ingress.yaml

```diff
@@ -18,10 +18,6 @@ spec:
   rules:
     - http:
         paths:
-          - path: /v1/workResults
```

Comment:
Found this information which seems to indicate this is related to load balancing [this](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.1/guide/ingress/spec/), but I think it might be good to consult someone with more knowledge on this before merging in

Reply:
I think we can delete this. Although I am not precisely sure what it does, maybe it is fixing these subscription errors we were getting in staging a few months ago?

Reply:
Correct! There is a special service that is only returning an error code that keeps retrying in case the subscription is not ready yet.